training.md

```diff
@@ -739,3 +739,26 @@ so our total batch size divided by the total number of chips cannot drop below
 $$\frac{4 \alpha^2}{F \cdot M_X \cdot M_Y},$$
 
 as we had derived above.
+
+### Appendix D - Unifying the "compute-bound" rules for various parallelism schemes
+
+To make it easier to remember, we can cast the various "compute-bound" rules from this section into a unified form which also highlights their physical origin. This form is:
+\begin{equation}
+\frac{\text{Effective matrix dimension}}{\text{Effective sharding-axis size}} \geq \frac{\text{Accelerator FLOPS/s}}{\text{Relevant bandwidth}}
```

Comment:
What does effective matrix dimension mean? In general, what does effective mean?